src/Codeception/Module/Phalcon.php

```diff
@@ -671,4 +670,15 @@ private function getApplicationDomainRegex()
 
         return '/^' . str_replace('.', '\.', $domain) . '$/';
     }
+
+    /**
+     * @return string
+     */
+    private function getMemorySessionForVersion() {
+        if (strpos(\Phalcon\Version::get(), '3.4') !== false) {
+            return PhalconConnector\MemorySession34x::class;
```

Comment:
This class will work with Phalcon 4.x too, right?
It would be better to use [version_compare](http://php.net/manual/en/function.version-compare.php)
`version_compare(\Phalcon\Version::get(), '3.4.0') >= 0)`

Reply:
It's unclear if it does work with 4.x I'll compile and test it to check. Good point though.

Reply:
It is more likely to work than the old class.

Reply:
yes :D probability says it is still if I don't need heuristics I won't use them :) 10 mins :) and we know it for sure I already changed it. I just verify it

Reply:
@Naktibalda currently there is not even an Session adapter interface in branch 4.0.x so let @sergeyklay decide how to proceed :). Maybe it's not consolidated atm.